theme/blog-btn.scss

```
@@ -7,6 +7,11 @@
   // pill. Block + line-height centering (the icon is absolutely positioned,
   // so no flexbox is needed) lets text-overflow clip it with an ellipsis;
   // `max-width` caps the pill at its container so the clip actually engages.
+  //
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)